packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/confirm-delete-database/confirm-delete-database.routing.js

```diff
@@ -0,0 +1,37 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.databases.dashboard.general-information.confirm-delete',
+    {
+      url: '/confirm-delete',
+      views: {
+        modal: {
+          component: 'ovhManagerPciProjectDatabaseConfirmDelete',
+        },
+      },
+      params: {
+        database: null,
+        linkedServices: [],
+      },
+      layout: 'modal',
+      resolve: {
+        breadcrumb: () => null,
+        database: /* @ngInject */ ($transition$) =>
+          $transition$.params().database,
```

Comment:
I think if you try to access directly this page via the browser navigation search it will fails.
You have to write a `redirectTo` function to check if database object is here or not.

There is some sample usage into PCI for this.
Need a help don't hesitate

Reply:
> I think if you try to access directly this page via the browser navigation search it will fails.

It will work. In that case `redirectTo` it not mandatory.